### ISSUE TITLE
Just a feature

### ISSUE THREAD

**Issue by LinkedHK** [spacecowboy/NoNonsense-FilePicker#81]
if you 've used the storage framework. it has the feature browsing by pictures/ videos(media files). It will greatly improve the UX, but i have no time  to get my hands on it. Please look into it.

**Comment by spacecowboy** [comment]
I'm not sure what you mean. Do you mean a new kind of layout?

**Comment by LinkedHK** [comment]
Correct. I mean navigation drawer which contains elements which can facilitate user's navigation. 
![image](https://cloud.githubusercontent.com/assets/2436637/15286140/c52f9166-1b8d-11e6-8a8b-c5e6ca4dcf14.png)
It sounds like rewriting of storage access framework . so the decision is up to you. I might  dedicate the time in my schedule to give you a hand.
